### DISCUSSION TITLE
showXcodeLog on server side

### DISCUSSION THREAD

**Discussion by luprochazka-cen63872** [appium/appium#19642] [Q&A] [answered]
Hi, I use showXcodeLog:true from a client side, is there a possibility to activate this on server side? I use Appium 2.4.1 and xcuitest@5.13.2. Thank you.

**Comment by KazuCocoa** [comment]
What do you mean the "server side"? The caps let appium print `xcodebuild` output as Appium server log.

**Reply by luprochazka-cen63872** [reply]
Yes, I know. But I use showXcodeLog in capabilities in RobotFramework like this:

`    Open Application  ${APPIUM_SERVER_URL}
    ...  platformName=IOS
    ...  deviceName=${NAZEV_ZARIZENI}
    ...  udid=${UDID}
    ...  automationName=XCUITest
    ...  bundleId=${bundle_id}
    ...  newCommandTimeout=4000
    ...  noReset=false
    ...  fullReset=false
    ...  adbExecTimeout=40000
    ...  appWaitForLaunch=true
    ...  showXcodeLog=true
    ...  orientation=LANDSCAPE`

I want to set this in config file for starting Appium server as global setting.

**Reply by KazuCocoa** [reply, marked as answer]
`--default-capabilities` would help. https://appium.io/docs/en/2.4/cli/args/

**Reply by luprochazka-cen63872** [reply]
Thank you. It worked:
`
{    "server": {
        "port": 4723,
        "base-path": "/wd/hub/",
        **"default-capabilities": {"appium:showXcodeLog": true}**,
        "use-plugins": ["appium-dashboard", "device-farm"],
        "plugin": {
            "device-farm": {
                "platform": "both"
            },
            "appium-dashboard": {
                "enableLiveVideo": "false"
            }
        }
    }
}

`
